config/redirects.js

```diff
@@ -2054,6 +2030,39 @@ module.exports = [
      to: '/applications/guides/register-spa'
    },
    {
+    from: [
+      '/libraries/lock/v9/customization',
+      '/libraries/lock/v9/configuration',
+      '/libraries/lock/v10/customization',
+      '/libraries/lock/v10/configuration'
+    ]
+    to: '/libraries/lock/v11/configuration'
+    },
+    {
+      from: [
+        '/libraries/lock/v10/popup-mode',
+        '/libraries/lock/v10/authentication-modes',
+        '/libraries/lock/v11/popup-mode'
+      ]
+      to: '/libraries/lock/v11/authentication-modes'
+    },
+    {
+      from: [
+        '/libraries/lock/v9',
+        '/libraries/lock/v9/display-modes',
+        '/libraries/lock/v9/configuration#container-string-',
+        '/libraries/lock/v9/types-of-applications',
+        '/libraries/lock/v10'
+      ]
+      to: '/libraries/lock/v11'
+    },
+    {
+      from: [
+        '/libraries/auth0js/v7',
+        '/libraries/auth0js/v8'
+      ]
+      to: '/libraries/auth0js/v9'
+    },
```

Comment:
Can you recheck this? It's not building and I thought I removed the problematic commas, but it doesn't appear to be the case.